.cursor/worktrees.json

```diff
@@ -0,0 +1,3 @@
+{
+  "setup-worktree": ["yarn install", "yarn build"]
```

Comment:
What do you think about just "yarn setup" ? That way it also does a clean build for safety, although it does take a bit longer

Reply:
Ok just did a manual git worktree setup.
The root cause is that you cannot run any yarn script before launching `yarn install`.
So, given `"setup": "yarn && yarn clean && yarn build --skip-nx-cache"` vs a new worktree:
- `yarn` is not relevant since we need to run it at top level first
- `yarn clean` is not relevant since the new worktree starts fresh (fs wise)
- `yarn build --skip-nx-cache` > `yarn build` is sufficient since first run